src/tracks/track.ts

```diff
@@ -142,16 +145,17 @@ export default class Track {
     }
   }
 
+  onDataLoaded(data?: any) {
+    if (this.isLoading) this.isLoading = false;
+  }
+
```

Comment:
is there supposed to be validation on the presence of data to set loading status?

Reply:
Track is an abstract class. The parameter is included to show its signature